client/src/components/CredentialsInputBox/CredentialsInputBox.css

```diff
@@ -0,0 +1,24 @@
+.cred-layout {
```

Comment:
Change to `CredentialsInputBox.modules.css` and make any necessary changes that use this class

Reply:
Refer to the comment made on the first CSS file that I commented on

Reply:
Fixed (Nestor): changed to .module